src/common/hooks/useRestoreToken.tsx

```diff
@@ -44,34 +44,66 @@ export const useRestoreToken = (
   const [errorMessage, setErrorMessage] = useState<string>();
   const [state, setState] = useState<ContractFunctionState>("UNINITIALIZED");
 
+  const sendToTitleEscrow = async (previousBeneficiary: string, previousHolder: string): Promise<void> => {
```

Comment:
Now that there is two huge functions `sendToTitleEscrow` and `deployAndSendToTitleEscrow`, you will likely want to refactor them outside of the hook. This way you can read this hook easier when you can see the function is `restoreToken` and that it either call `sendToTitleEscrow` or `deployAndSendToTitleEscrow`.

Reply:
ok have changed it.